pkg/ctl/cmdutils/configfile.go

```diff
@@ -126,8 +126,28 @@ func NewGitopsMetadataLoader(cmd *Cmd) ClusterConfigLoader {
 	l := newCommonClusterConfigLoader(cmd)
 
 	l.validateWithoutConfigFile = func() error {
+		meta := l.ClusterConfig.Metadata
+		if meta.Name == "" {
+			return ErrMustBeSet("--cluster")
+		}
+		return nil
+	}
+
+	return l
+}
+
+// NewInstallMetadataLoader handles loading of clusterConfigFile vs using flags for install commands
+func NewInstallMetadataLoader(cmd *Cmd) ClusterConfigLoader {
```

Comment:
Let's call it `NewInstallFluxLoader`, just like for each of other commands that don't use supposed-to-be-generic metadata loader... This package does need some thought going forward, it's a half-baked idea of a shared abstraction that clearly has a few issues.

Reply:
Sure, I will rename the GitOps one too for consistency